src/node_v8.cc

```diff
@@ -20,6 +20,7 @@
 // USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 #include "node.h"
+#include "node_internals.h"
```

Comment:
Is this include and the one in uv.cc necessary?

Reply:
Yes. need to add `node_internals.h` for node_v8.cc and uv.cc